Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Step 1 - Build Front End Application
-FROM node:16 AS build
+FROM node:14 AS build
```

Comment:
Why downgrade?

Reply:
Good spot! I meant to upgrade this again. I couldn't build it on my machine using node 16. Some M1 issue seemingly.